water.mss

```diff
@@ -1,4 +1,4 @@
-@water-text: #6699cc;
+@water-text: darken(#6699cc, 10%);
```

Comment:
Now that the color testing is over, I would simply change it to a proper hex value (#4D80B3).